libs/core/src/lib/mega-menu/mega-menu-item/mega-menu-item.component.ts

```diff
@@ -92,33 +94,21 @@ export class MegaMenuItemComponent implements AfterContentInit, OnDestroy, Defau
     constructor(
         private elRef: ElementRef,
         private menuKeyboardService: MenuKeyboardService,
-        private changeDetectionRef: ChangeDetectorRef
-    ) {}
+        private changeDetectionRef: ChangeDetectorRef,
+        @Optional() private rtlService: RtlService
+    ) {
+        if (rtlService) {
```

Comment:
- Unsubscribe
- Can we move this to separate method?

Reply:
done